include/bluetooth/mesh/light_ctrl.h

```diff
@@ -183,6 +184,44 @@ enum bt_mesh_light_ctrl_coeff {
 	BT_MESH_LIGHT_CTRL_COEFF_KPU = BT_MESH_PROP_ID_LIGHT_CTRL_REG_KPU,
 };
 
+/** @brief Get the sensor format for the given Light Lightness Control property ID.
+ *
+ *  @param[in] id Light Lightness Control property or coefficient ID.
```

Comment:
The coefficients are properties, no need to doubly specify. (As in, the are all listed under "6.2.4 Light LC Property states", no distinction is made in the spec. I see no reason to make a somewhat arbitrary distinction here)